frontends/precinct-scanner/src/app_root.tsx

```diff
@@ -544,17 +546,29 @@ export function AppRoot({
     return <InsertUsbScreen />;
   }
 
-  if (needsToReplaceBallotBag && scannerStatus.state !== 'accepted') {
+  // when we remove the poll worker card
+  if (showReplaceBallotBagScreen && !isPollWorkerAuth(auth)) {
+    setShowReplaceBallotBagScreen(false);
+  }
+
+  if (
+    (needsToReplaceBallotBag && scannerStatus.state !== 'accepted') ||
+    showReplaceBallotBagScreen
+  ) {
     return (
       <ReplaceBallotBagScreen
         scannedBallotCount={scannerStatus.ballotsCounted}
         pollWorkerAuthenticated={isPollWorkerAuth(auth)}
-        onComplete={() =>
+        onComplete={() => {
+          // pin the state to be in "replacing ballot bag" mode
+          // so that, when we update the state, we don't immediately
+          // unmount this component.
+          setShowReplaceBallotBagScreen(true);
```

Comment:
note to self: this flag is only applicable to the original, forced ballot bag replacement path

Reply:
maybe this could cause a flicker, not sure

Reply:
did not see a flicker. I think this shouldn't, since this just flips the second of the OR'ed conditions for showing the replace-bag-screen.